### ISSUE TITLE
Push Dashboard - Search by custom properties confusing

### ISSUE THREAD

**Issue by sagivf** [driftyco/ionic-cloud#52]
Although there is no bug and you should use `details.email` and not `custom.email`,
this is very confusing.

Also I would add a spinner when searching (can also be confusion)

## **Using `custom.email`**
![image](https://cloud.githubusercontent.com/assets/1857466/18708012/67f1c13a-8001-11e6-877e-e3b74e431735.png)

## **Using `details.email`**
![image](https://cloud.githubusercontent.com/assets/1857466/18708028/7fe00324-8001-11e6-8b03-0fa44a76b75d.png)

**Comment by ericb** [comment]
Hey @sagivf 👋 

Was this confusing because you collect a secondary email and store it in the custom properties, or more in the way that you had once saved an email property under custom, and then realized it was already in details?

The spinner I definitely agree with.

**Comment by sagivf** [comment]
Hi @ericb :smile: 
I'm actually not sure how the custom email property got there. I thought It was something you guys add. I can't find anything in my code that should be doing it, but I might have saved it there at some point.

I don't see it here either on any of the users:
![image](https://cloud.githubusercontent.com/assets/1857466/18711040/c7f72fca-8010-11e6-884d-7dfbe132c5fd.png)

Do you not see it on other users/apps?

It's worth checking, but the simplest thing would be to keep the `details.xxx` properties sorted to the top of the list...

**Comment by sagivf** [comment]
Thing is I've trip over this a few times and always think there is something wrong in the system till I realise what's really wrong...

**Comment by sagivf** [comment]
Oh and I just read whats you wrote again - no, I'm not collecting a second email. not sure how this property got there...

**Comment by ericb** [comment]
Ah, okay. Yeah, those fields are populated by custom properties you've saved at *any* point for an app. So if you had a single save early on that had email in the custom properties, it would have stored that field for future lookup.

There's a number of things we may do with that system in the future, but one of the things we could do in the short term is to provide a way to delete the lookup fields (this wont remove data from users, only remove it from the lookup table so that it would not show in the dropdown list).

This way fields that may be added during development can be cleaned up when you know you're not using them anymore.

**Comment by sagivf** [comment]
Yep - that sounds good 👍  feel free to rename the issue or close it. I would however reorder or emphasis details vs custom sections - it's easy to not notice which one you are on.

**Comment by ericb** [comment]
I'll keep this open for now, since I believe both of those suggestions to be a good thing for us 👍
